book/content/open-source-comms/issue-tracking.md

```diff
@@ -0,0 +1,80 @@
+- Contributors can subscribe to notifications or updates from only the repositories that interest them;
+- It feels like "divide and conquer", more people are working on more aspects to move the project as a whole forward.
+
+#### Case study: mybinder.org
+
+[mybinder.org](https://mybinder.org) is a platform facilitating users to easily share reproducible analyses and computational environments with one another in [Jupyter Notebooks](https://jupyter-notebook.readthedocs.io/en/stable/) via the cloud.
+This project is spread across a number of different repositories, each one an individual tool that can be used in isolation from the others.
+These are:
+
+- [repo2docker](https://github.com/jupyter/repo2docker),
+- [JupyterHub for Kubernetes](https://github.com/jupyterhub/zero-to-jupyterhub-k8s),
+- [BinderHub](https://github.com/jupyterhub/binderhub).
+
+Each of these repositories contains hundreds of issues tracking on-going work being performed by the community and scoping future directions for each project to take.
+
+Can you imagine trying to combine all of these issues into one place?
+It would become very difficult, if not impossible, for someone to find what they are looking for and would require a very clever tagging schema plus instructions for filtering by tag.
+
+In the Project Binder team's experience, most community members contribute to just one or maybe two of these projects.
+So having consolidated access to all the issues for all the working parts is not a high priority for their community.
+
+They find that having distributed issue tracking allows those members of the community who may only work with JupyterHub to comfortably contribute without needing to be familiar with everything that goes into running [mybinder.org](https://mybinder.org).
+
+### Centralised Issue Repository
+
+With a big project, it can be tempting to collate all your issues into one place for the sake of easier management.
+However in terms of engaging your community, this can be problematic.
+If your issues are elsewhere this can create a lot of barriers to entry for community members, such as:
+
+- Issues are more difficult to discover;
+- If they are hosted on another platform, that's another thing community members need to learn how to use;
+- Issues are separate to the code they are referencing.
+
+A very large impact on the community of having a separate issue repository is that when people visit your code repository, it looks like an inactive project because there are no issues or conversations going on where the code is hosted.
+This may cause community members to believe that the code is no longer being actively developed/maintained/supported and may choose to use another codebase or software package.
+
+## Comparative Table
+
+| Feature | Single Issue Repo | Issue Tracker per Repo |
+| :--- | :---: | :---: |
+| Global issue search | ✅ | |
+| Hosted by the same Platform | ❓(not guaranteed) | ✅ |
```

Comment:
I'm not sure I understand this row. My expectation was that a single repo would (by construction) be hosted on the same platform. Where are for a distributed model each issue tracker could be on a different platform.

Reply:
So the case I came across in the wild is a GitHub org with lots of repos but _all_ of the issues are hosted on Atlassian. It's super confusing and difficult to make connections between issues and repos. I didn't intend for "centralised" to mean a single repo here.